app/Http/Controllers/UserController.php

```diff
@@ -13,7 +14,9 @@ class UserController extends Controller
      */
     public function index()
     {
-        //
+        $listUser = User::paginate(3);
+        
+        return view('admin.user.list')->with(compact('listUser'));
```

Comment:
```suggestion
        return view('admin.user.list')->with(compact('listUser'));
```  return view('admin.user.list', compact('listUser'));